packages/twenty-front/src/modules/object-record/record-filter/utils/getCorrespondingSubfieldFromLabel.ts

```diff
@@ -0,0 +1,40 @@
+export const getAddressSubField = (label: string) => {
+  switch (label) {
+    case 'Address 1':
+      return 'addressStreet1';
+    case 'Address 2':
+      return 'addressStreet2';
+    case 'City':
+      return 'addressCity';
+    case 'State':
+      return 'addressState';
+    case 'Country':
+      return 'addressCountry';
+    case 'Post Code':
+      return 'addressPostcode';
+    default:
+      throw new Error(`Unknown address subfield label ${label}`);
+  }
```

Comment:
think you can update `SETTINGS_FIELD_TYPE_CONFIGS` and add SubLabels keys for composite types and use that

Reply:
Nice idea !